cmd/airflow.go

```diff
@@ -60,7 +60,6 @@ var (
 	envExport              bool
 	noBrowser              bool
 	compose                bool
-	conflictTest           bool
```

Comment:
I don't see anywhere in our code that sets this variable. It used to exist at one time as a flag, but doesn't anymore. This would always evaluate to `false`, so we remove the code that branches if this value is `true`.